### PR TITLE
Add option to keep the existing b-factor in the input PDB

### PR DESCRIPTION
**Pull request by amjjbonvin**
You are about to submit a new Pull Request. Before continuing make sure you read the [contributing guidelines](CONTRIBUTING.md) and that you comply with the following criteria:

- [ ] You have sticked to Python. Please talk to us before adding other programming languages to HADDOCK3
- [X] Your PR is about CNS
- [ ] Your code is well documented: proper docstrings and explanatory comments for those tricky parts
- [ ] You structured the code into small functions as much as possible. You can use classes if there is a (state) purpose
- [X] Your code follows our coding style
- [ ] You wrote tests for the new code
- [X] `tox` tests pass. *Run `tox` command inside the repository folder*
- [X] `-test.cfg` examples execute without errors. *Inside `examples/` run `python run_tests.py -b`*
- [ ] PR does not add any dependencies, unless permission granted by the HADDOCK team
- [ ] PR does not break licensing
- [ ] Your PR is about writing documentation for already existing code :fire:
- [ ] Your PR is about writing tests for already existing code :godmode:

---

Adds the option to keep the existing b-factor in the input PDB files by setting `set_bfactor=false` in topoaa.
b-factors of unknown atoms are assigned the average b-factor of the known atoms per residue

### CODE REVIEW

File: src/haddock/modules/scoring/emscoring/defaults.yaml [hunk 1]
```diff
@@ -1,6 +1,14 @@
...
 # Group scoring
 #
+per_interface_scoring:
+  default: false
+  type: boolean
+  title: Interchains scoring
+  short: Output scores per interface in the PDB header.
+  long: With this option true, the interface of all combinations of chains will be analysed and the corresponding HADDOCK scores (without restraints terms) and their components (Eelec, Evdw, Edesol, BSA) will be written to the header of each PDB file.
+  group: 'scoring'
+  explevel: expert
```
Comment (mgiulini) [inline src/haddock/modules/scoring/emscoring/defaults.yaml:11]: this does not belong to this PR, right?

**Review by mgiulini — APPROVED** [review]
works!
I think there are overlaps with PR #896 but hopefully merging both does not cause conflicts

**Comment by amjjbonvin** [comment]
Looks like I made a mess of those pull requests….

**Comment by amjjbonvin** [comment]
Should now be fine - please review

**Review by mgiulini — APPROVED** [review]
